ui/raidboss/raidboss_config.ts

```diff
@@ -1260,18 +1265,34 @@ class RaidbossConfigurator {
     // Should we show them in the parser language instead?
     const lang = this.base.lang;
 
-    const getRegex = (baseField: 'regex' | 'netRegex') => {
-      const regex = trig[baseField];
+    const getRegex = () => {
+      const regex = trig.regex;
       if (regex === undefined)
         return;
       return Regexes.parse(translateRegex(regex, lang, set.timelineReplace));
     };
 
+    const getNetRegex = () => {
+      const regex = trig.netRegex;
+      if (regex === undefined)
+        return;
+
+      if (trig.type === undefined) {
+        if (!(regex instanceof RegExp))
+          return /!!Bad Trigger!!/;
```

Comment:
It's to show on config page, console.log will be ignored and throwing exception will cause whole config page not working.

Reply:
I think in general it won't show at all, as most people won't have this turned on.  What about just `return;` so that it's `undefined` as if it were missing? I think I just don't like special strings like this.